src/connector.rs

```diff
@@ -388,13 +398,15 @@ impl EthConnectorContract {
         self.ft
             .internal_withdraw(&predecessor_account_id, args.amount);
         // Save new contract data
-        self.save_contract();
+        self.save_ft_contract();
         sdk::return_output(&res[..]);
     }
 
     /// Return total supply of NEAR + ETH
     pub fn ft_total_supply(&self) {
         let total_supply = self.ft.ft_total_supply();
+        // TODO: To make it match the standard, it has to be a string instead of an integer.
+        //    So we need to wrap the `100` into a string like `"100". This is due to JSON int limits.
         sdk::return_output(&total_supply.to_string().as_bytes());
```

Comment:
I thought to match the standard it has to return a string. If we use U256 integers internally and return the string (UTF-8 bytes), is there any problem with it?